pkg/bindings/down/play.go

```diff
@@ -0,0 +1,39 @@
+package down
+
+import (
+	"context"
+	"net/http"
+	"os"
+
+	"github.com/containers/podman/v4/pkg/bindings"
+	"github.com/containers/podman/v4/pkg/domain/entities"
+	"github.com/sirupsen/logrus"
+)
+
+func KubeDown(ctx context.Context, path string) (*entities.PlayKubeReport, error) {
```

Comment:
This moves KubeDown from the previous package into here, then redirects the old function to this new one.

However, the old one returned PlayKubeReport (which I suppose we cannot change to PlayKubeTeardown without breaking API compatibility)